pandas/tests/test_sorting.py

```diff
@@ -7,7 +7,7 @@
 from numpy import nan
 import pytest
 
-from pandas.compat import PY2
+from pandas.compat.numpy import _np_version_under1p14
```

Comment:
remove all actual PY2 changes from this PR, this should only be about changing the CI itself.

Reply:
Of course I'm not aiming to change code here, but `test_unsortable` below would fail otherwise. The current regex is not general enough (the skipif for `PY2` is a cop-out IMO), and *will* need to be fixed for a passing CI.